Docs/source/usage/parameters.rst

```diff
@@ -1919,6 +1919,22 @@ In-situ capabilities can be used by turning on Sensei or Ascent (provided they a
     Default is ``<diag_name>.fields_to_plot = Ex Ey Ez Bx By Bz jx jy jz``.
     Note that the fields are averaged on the cell centers before they are written to file.
 
+* ``<diag_name>.particle_fields_to_plot`` (list of `strings`, optional)
+   Names of per-cell averages of particle properties to calculate and output. Default is none. Parser functions for these field names are specified by ``<diag_name>.particle_fields.<field_name>(x,y,z,ux,uy,uz)``.
+
+* ``<diag_name>.particle_fields_species`` (list of `strings`, optional)
+         Species for which to calculate ``particle_fields_to_plot``.
+         The default is a list of all of the available particle species.
+
+* ``<diag_name>.particle_fields.<field_name>(x,y,z,ux,uy,uz)`` (parser `string`)
+   Parser function to be calculated for each particle and averaged per cell. The field written to the plotfile is
```

Comment:
Also works with openPMD and all other full diagnostics (Ascent/SENSEI), I think :)
```suggestion
   Parser function to be calculated for each particle and averaged per cell. The field written is
```
Can you please test it works with openPMD?

Reply:
I've checked that the new fields are available in the openPMD files, and the values are correct. Should I add an OpenPMD test to CI?

One thing I noticed was that the arrays I get from `ts.get_field('field=field')` and `ad0['boxlib','field'][:,:,0]` are transposes of one another in 2D (haven't checked 3D). But this seems to be the same even for old fields, so I assume it's normal behavior?

Reply:
CI: great idea!

transpose: that's possible, yes.